code/modules/reqs/supply.dm

```diff
@@ -441,11 +480,11 @@ GLOBAL_LIST_INIT(blacklisted_cargo_types, typecacheof(list(
 					playsound(SSshuttle.supply.return_center_turf(), 'sound/machines/buzz-two.ogg', 50, 0)
 				else
 					playsound(SSshuttle.supply.return_center_turf(), 'sound/machines/elevator_move.ogg', 50, 0)
-					SSshuttle.moveShuttle("supply", "supply_away", TRUE)
+					SSshuttle.moveShuttle(shuttle_id, away_id, TRUE)
 			else
-				var/obj/docking_port/D = SSshuttle.getDock("supply_home")
+				var/obj/docking_port/D = SSshuttle.getDock(home_id)
 				playsound(D.return_center_turf(), 'sound/machines/elevator_move.ogg', 50, 0)
-				SSshuttle.moveShuttle("supply", "supply_home", TRUE)
+				SSshuttle.moveShuttle(shuttle_id, home_id, TRUE)
```

Comment:
will away require additional mapping because it should be sending to transitspace instead

Reply:
fixed